tests/src/TestShapeMocks.cpp

```diff
@@ -49,7 +49,7 @@ TEST_CASE("MockShape", "[mocks::MockShape]")
   GIVEN("A mock shape with function objects returning values")
   {
     mocks::MockShape shape;
-    size_t area_f_count = 0;
+    int area_f_count = 0;
```

Comment:
'Twas an attempt to silence MSVC warnings. It still complains about Catch so I've not won that battle (yet).